src/preprocess/scale.py

```
@@ -0,0 +1,18 @@
+
+def set_surface_area_to_one(mesh: tm.Trimesh) -> tm.Trimesh:
+    """ Scales a copy of the mesh to have a surface area of 1.
+
+    Args:
+        mesh: Input mesh to scale.
+
+    Returns:
+        A new mesh with surface area normalized to 1.
+    """
+    if mesh.area <= 0:
+        raise ValueError(f"Mesh has invalid surface area: {mesh.area}. Expected positive value.")
```

Ruff (0.13.3)

Avoid specifying long messages outside the exception class

(TRY003)